report/23_Process-CI-CD-Description.md

```diff
@@ -1,2 +1,13 @@
-A complete description of stages and tools included in the CI/CD chains.
-That is, including deployment and release of your systems.
+All CI/CD in the project is achieved using GitHub actions. The different steps of the CI/CD process have been split into workflow files with separate responsibilities:
+- ```dotnetcore.yml``` runs unit tests on the project whenever code is pushed.
+- ```e2e.yml``` runs end-to-end tests when a pull request to master is made.
+- ```sonarPR.yml``` provides a report with code analysis from SonarCloud whenever a pull request to master is made. This report is put on the pull request for easy access.
```

Comment:
I would add that it can be used to block merging if CodeCoverage is too low or issues come up. We have not set this part up, but it would be useful.

Reply:
Can't find a good way to write this, since it is too much information to have in the list of scripts, and it would be awkward to write in the other place where sonarPR is mentioned. Can we just leave it out?